nodejs/week1/src/backend/server.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.get("/", (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+
+const server = app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`);
+});
+
+// to display all the meals
+const allMeals = require("./routes/meals");
+app.get("/meals", allMeals);
+
+//to dispaly a random meal
+const randomMeal = require("./routes/meal");
+app.get("/randomMeal", randomMeal);
```

Comment:
You should avoid capital letters in urls - /random-meal is more in line with the standard... 👍